plugins/ltac2/tac2core.ml

```diff
@@ -500,6 +500,14 @@ let () =
   define "constr_equal" (constr @-> constr @-> tac bool) @@ fun c1 c2 ->
   Proofview.tclEVARMAP >>= fun sigma -> return (EConstr.eq_constr sigma c1 c2)
 
+let () =
+  define "constr_conv" (constr @-> constr @-> tac bool) @@ fun c1 c2 ->
+  Proofview.tclENV >>= fun env ->
+  Proofview.tclEVARMAP >>= fun sigma ->
+  match (Reductionops.infer_conv env sigma c1 c2) with
```

Comment:
this is wrong, the env isn't correct

Reply:
I don't know what to do as `Proofview.Goal.env` would require `Control.enter` but this expects sth of type unit

Reply:
Is it better with enter_one ?

Reply:
use pf_apply (defined in this file)